README.md

```diff
@@ -338,10 +338,27 @@ As of release 1.2.0, the following are not implemented and will be covered in fu
 
 ### Tests
 
-This project contains two types of test. Unit tests and integration tests. The integration tests depend on a git submodule being present. To clone the repo with submodules `git clone git@github.com:ably/ably-go.git --recurse-submodules` 
+This project contains two types of test. Test which use the `ablytest` package and tests which dont. 
 
-- Unit tests can be run with `go test -tags=unit ./...`
-- Integration tests can be run using `scripts/test.sh`.
+#### Unit tests
```

Comment:
Headings should be followed by a blank line (in my opinion anyway)

Reply:
Thank you @Morganamilo blank line is now added.